src/core/inlines.js

```diff
@@ -30,6 +30,18 @@ const inlineVariable = /\B\|\w[\w\s]*(?:\s*:[\w\s&;<>]+)?\|\B/; // |var : Type|
 const inlineCitation = /(?:\[\[(?:!|\\|\?)?[A-Za-z0-9.-]+\]\])/; // [[citation]]
 const inlineExpansion = /(?:\[\[\[(?:!|\\|\?)?#?[\w-.]+\]\]\])/; // [[[expand]]]
 const inlineAnchor = /(?:\[=[^=]+=\])/; // Inline [= For/link =]
+const inlineElement = /(?:\[\^[A-Za-z]+\^\])/; // Inline [^element^]
```

Comment:
```suggestion
const inlineElement = /(?:\[\^[A-Za-z]+(?:-[A-Za-z]+)?\^\])/; // Inline [^element^]
```
or simpler and the one accepting potentially  bad input:
```suggestion
const inlineElement = /(?:\[\^[A-Za-z-]+\^\])/; // Inline [^element^]
```